xml/ha_concepts.xml

```diff
@@ -607,122 +607,99 @@
+         each. The <systemitem class="daemon">pacemaker-based</systemitem>
+         daemon takes care of reading and writing cluster configuration and
+         status.</para>
+       </listitem>
+      </varlistentry>
      <varlistentry xml:id="vle.dc">
       <term>Designated Coordinator (DC)</term>
       <listitem>
        <para>
-        One CRM in the cluster is elected as DC. The DC is the only entity
+        The DC is elected from all nodes in the cluster after a membership change
+        (nodes joining or leaving the cluster). The DC is the only entity
         in the cluster that can decide that a cluster-wide change needs to
-        be performed, such as fencing a node or moving resources around. The
-        DC is also the node where the master copy of the CIB is kept. All
+        be performed, such as fencing a node or moving resources around. All
         other nodes get their configuration and resource allocation
-        information from the current DC. The DC is elected from all nodes in
-        the cluster after a membership change.
+        information from the current DC.
        </para>
       </listitem>
      </varlistentry>
-     <varlistentry xml:id="vle.pe.and.te">
-      <term>Policy Engine (PE)</term>
+     <varlistentry xml:id="vle.pe">
+      <term>Policy Engine</term>
       <listitem>
        <para>
-        Whenever the Designated Coordinator needs to make a cluster-wide
-        change (react to a new CIB), the Policy Engine calculates the next
-        state of the cluster based on the current state and configuration.
-        The PE also produces a transition graph containing a list of
-        (resource) actions and dependencies to achieve the next cluster
-        state. The PE always runs on the DC.
+        The policy engine always runs on the DC. The engine is implemented as
+        <systemitem class="daemon">pacemaker-schedulerd</systemitem> daemon. It
+        schedules all actions that are needed whenever the CIB changes
+        changes and calculates the next state of the cluster,
+        based on the current state and configuration.
        </para>
       </listitem>
      </varlistentry>
-     <varlistentry xml:id="vle.lrm">
-      <term>Local Resource Manager (LRM)</term>
-      <listitem>
-       <para>
-        The LRM calls the local Resource Agents (see
-        <xref linkend="sec.ha.architecture.layers.resources"/>) on behalf of
-        the CRM. It can thus perform start / stop / monitor operations and
-        report the result to the CRM. The LRM is the authoritative source
-        for all resource-related information on its local node.
-<!--In &productname;, the LRM is
-        implemented as daemon, interacting directly with resource agents.
-        The daemon is not cluster-aware and presents a common interface to
-        the supported resource types.-->
-       </para>
-      </listitem>
-     </varlistentry>
```

Comment:
Pacemaker-attrd is probably a less important daemon, since it's also missing in the graph :-)  Probably some day we should eventually add it also.

Reply:
Added as a comment for future versions. :)